fastlane/Fastfile

```diff
@@ -711,130 +700,130 @@ GH_ORG_NAME="woocommerce"
   ########################################################################
-  private_lane :alpha_code_signing do |options|
+  private_lane :alpha_code_signing do |_options|
     match(
-      type: "enterprise",
-      team_id: get_required_env("INT_EXPORT_TEAM_ID"),
+      type: 'enterprise',
+      team_id: get_required_env('INT_EXPORT_TEAM_ID'),
       readonly: true,
-      app_identifier: ["com.automattic.alpha.woocommerce"])
+      app_identifier: ['com.automattic.alpha.woocommerce']
+    )
   end
 
-  private_lane :appstore_code_signing do |options|
+  private_lane :appstore_code_signing do |_options|
     match(
-      type: "appstore",
-      team_id: get_required_env("EXT_EXPORT_TEAM_ID"),
+      type: 'appstore',
+      team_id: get_required_env('EXT_EXPORT_TEAM_ID'),
       readonly: true,
-      app_identifier: ["com.automattic.woocommerce"])
+      app_identifier: ['com.automattic.woocommerce']
+    )
   end
 
   ########################################################################
   # Helper Lanes
   ########################################################################
-  desc "Get a list of pull requests from the `milestone`"
-  lane :get_pullrequests_list do | options |
-    get_prs_list(repository:GHHELPER_REPO, milestone:"#{options[:milestone]}", report_path:"#{File.expand_path('~')}/wcios_prs_list.txt")
+  desc 'Get a list of pull requests from the `milestone`'
+  lane :get_pullrequests_list do |options|
+    get_prs_list(repository: GHHELPER_REPO, milestone: (options[:milestone]).to_s,
+                 report_path: "#{File.expand_path('~')}/wcios_prs_list.txt")
   end
 
-  desc "Run release preflight checks"
-  lane :release_preflight do | options |
-      configure_validate()
+  desc 'Run release preflight checks'
+  lane :release_preflight do |_options|
+    configure_validate
   end
 
-  desc "Run localization only"
-  lane :update_localization do | options |
-    ios_localize_project()
+  desc 'Run localization only'
+  lane :update_localization do |_options|
+    ios_localize_project
   end
 
   def write_file(path, contents)
-    begin
-      FileUtils.rm_f(path)
-      file = File.new(path, "w")
-      file.write(contents)
-    rescue IOError => e
-      UI.crash(e)
-    ensure
-      file.close unless file.nil?
-    end
+    FileUtils.rm_f(path)
+    file = File.new(path, 'w')
+    file.write(contents)
+  rescue IOError => e
+    UI.crash(e)
+  ensure
+    file.close unless file.nil?
   end
 
   def ios_locales
-    "ar de-DE en-US es-ES fr-FR he id it ja ko nl-NL pt-BR ru sv tr zh-Hans zh-Hant".split()
+    'ar de-DE en-US es-ES fr-FR he id it ja ko nl-NL pt-BR ru sv tr zh-Hans zh-Hant'.split
   end
 
   def glotpress_locales
-    "ar de es fr he id it ja ko nl pt-br ru sv tr zh-cn zh-tw".split()
+    'ar de es fr he id it ja ko nl pt-br ru sv tr zh-cn zh-tw'.split
   end
-
 end
 
-  def fastlane_directory()
-    File.expand_path File.dirname(__FILE__)
-  end
+def fastlane_directory
+  __dir__
+end
```

Comment:
This is beyond the scope of this PR and what RuboCop can auto correct, but, eventually, it would be good to have this and all the other methods returning constant values as actual constants 😄 

```suggestion
FASTLANE_DIRECTORY = __dir__
```

We'll get there 💪

Reply:
Good point and nice catch

Reply:
Actually probably fixing this one is worth it in this PR 🤔

Reply:
🤷‍♂️ up to you

Reply:
@mokagio Fixed in f6121fe

Feel free to give it a quick check and merge the PR yourself if all looks good.